### PR TITLE
Add missing debugsend and patch operation for NotificationHub resource

### PR DESCRIPTION
**Pull request by blocphan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json [hunk 1]
```diff
@@ -916,6 +974,60 @@
...
+          "NotificationHubs"
+        ],
+        "x-ms-examples": {
+          "debugsend": {
+            "$ref": "./examples/NotificationHubs/NotificationHubDebugSend.json"
+          }
+        },
+        "operationId": "NotificationHubs_DebugSend",
+        "description": "test send a push notification",
+        "parameters": [
```
Comment (jianghaolu) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json]: Missing parameter 'subscriptionId'

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubPatch.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+    "parameters": {
+      "namespaceName": "nh-sdk-ns",
+      "resourceGroupName": "sdkresourceGroup",
+      "notificationHubName": "sdk-notificationHubs-8708",
+      "api-version": "2017-04-01",
+      "subscriptionId": "29cfa613-cbbc-4512-b1d6-1b3a92c7fa40",
+      "properties": {
```
Comment (jianghaolu) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubPatch.json]: This should be "parameters"

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubDebugSend.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+        "namespaceName": "nh-sdk-ns",
+        "resourceGroupName": "5ktrial",
+        "notificationHubName": "nh-sdk-hub",
+        "api-version": "2017-04-01",
+        "subscriptionId": "29cfa613-cbbc-4512-b1d6-1b3a92c7fa40"
+    },
+    "responses": {
+        "200": {
```
Comment (jianghaolu) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubDebugSend.json]: "200" is not a valid response defined in the spec.

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubDebugSend.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{
+    "parameters": {
+        "namespaceName": "nh-sdk-ns",
+        "resourceGroupName": "5ktrial",
+        "notificationHubName": "nh-sdk-hub",
+        "api-version": "2017-04-01",
+        "subscriptionId": "29cfa613-cbbc-4512-b1d6-1b3a92c7fa40"
+    },
```
Comment (jianghaolu) [inline specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/examples/NotificationHubs/NotificationHubDebugSend.json]: Required parameter "parameters" missing.

File: specification/notificationhubs/resource-manager/readme.md [hunk 5]
```diff
@@ -24,6 +24,16 @@ To see additional help and options, run:
 ### Basic Information
 These are the global settings for the NotificationHubs API.
 
+directive:
```
Comment (jianghaolu) [inline specification/notificationhubs/resource-manager/readme.md]: The directives must be defined in the yaml section for the tag containing this definition

**Review by jianghaolu — CHANGES_REQUESTED** [review]

File: specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json (hunk 1 above)
Comment (blocphan) [reply specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-04-01/notificationhubs.json]: Fixed now

**Review by jianghaolu — APPROVED** [review]

**Pull request merged**